query/src/main/java/org/infinispan/query/clustered/ClusteredQueryInvoker.java

```diff
@@ -46,7 +47,7 @@
       this.asyncExecutor = asyncExecutor;
       this.rpcManager = cache.getRpcManager();
       this.myAddress = rpcManager.getAddress();
-      this.rpcOptions = rpcManager.getRpcOptionsBuilder(ResponseMode.SYNCHRONOUS).timeout(10000, TimeUnit.MILLISECONDS).build();
+      this.rpcOptions = new RpcOptions(DeliverOrder.NONE, 10000, TimeUnit.MILLISECONDS);
```

Comment:
nitpick: no idea why this line has the timeout hardcoded... 
can it be removed and use `rpcManager.getSyncRpcOptions()` below?

Reply:
Hmm I'm not sure either. @gustavonalle Any idea if this would have a detrimental effect?

Reply:
This will be configurable soon, don't worry about it